common/OpTestHMC.py

```diff
@@ -80,6 +97,41 @@ def check_kernel_signature(self):
             return "signed_kernel"
         return "unsigned_kernel"
 
+    def get_grub_file(self):
+        '''
+        Fetch the grub-file based on the distro
+        '''
+        if self.distro == 'rhel':
+            cmd = "rpm -ql grub2-ppc64le"
+            out = self.console.run_command(cmd)
+            for line in out:
+                if 'core.elf' in line:
+                    grub_filename = line
+            if not grub_filename:
+                self.fail("%s: Failed to get grub file" % (self.distro))
+        elif self.distro == 'suse' or self.distro == 'sles':
+            cmd = "rpm -ql grub2-powerpc-ieee1275"
+            out = self.console.run_command(cmd)
+            for line in out:
+                if 'grub.elf' in line:
+                    grub_filename = line
+            if not grub_filename:
+                self.fail("%s: Failed to get grub file" % (self.distro))
+        else:
+            raise self.skipTest("Test currently supported on SLES and RHEL releases only.")
+        return grub_filename
+
+    def check_grub_signature(self, grub_filename):
+        '''
+        Check whether the GRUB is signed or unsigned.
+        '''
+        cmd = "strings %s | tail -1" % grub_filename
+        output = self.console.run_command(cmd)
+        if "Module signature appended" in output[0]:
+            return "signed_grub"
```

Comment:
Why not return 'true' or 'false'?

Reply:
Yup and also how about changing the name to is_grub_signed() and return true/false?